src/stream/src/executor/sync_kv_log_store.rs

```diff
@@ -145,6 +152,12 @@ struct FlushedChunkInfo {
 }
 
 enum WriteFuture<S: LocalStateStore> {
+    Paused {
```

Comment:
Make add comments about the motivation of introducing the `Paused` item, and some details about the strategy. The strategy may include the condition to following actions
- pause when seeing an upstream data chunk
- pause when seeing an upstream checkpoint barrier
- unpause a paused write future when just consumed a chunk

Reply:
Updated